src/components/ModalPayment/SquareModal.tsx

```diff
@@ -175,14 +179,29 @@ const SquareModal = ({
                 <Checkbox
                   value="checkedA"
                   inputProps={{ 'aria-label': 'Checkbox A' }}
-                  onClick={checkAgreement}
-                  checked={isChecked}
+                  onClick={checkTermsAgreement}
+                  checked={isTermsChecked}
                 />
                 <span>
                   I agree with the <b>Terms & Conditions</b>
                 </span>
               </label>
             </div>
+            <div>
+              <label className={styles.termsAndConditions}>
+                <Checkbox
+                  value="checkedB"
+                  inputProps={{ 'aria-label': 'Checkbox B' }}
+                  onClick={checkSubscriptionAgreement}
+                  checked={isSubscriptionChecked}
+                />
+                <span>
+                  I'd like to receive email updates from Send Chinatown Love,
```

Comment:
Is this text too verbose? Maybe we can truncate to `I'd like to receive email updates from Send Chinatown Love` @nanxiy

Reply:
@jtmckibb wanted to have context around what people are signing up for -- I agree it could be shorter. up to you guys!

Reply:
I personally think that if we just say "I'd like to receive email updates" then people will automatically not check the box instinctually. 

We could do something more colloquial like "I'd like to stay in the loop and know when my merchant is paid, and when new merchants onboard"

Reply:
"Email updates" — I just automatically always uncheck. We can keep it short but also be annoying and do a pop-up and say "Are you sure? (insert sad puppe) We only email you things like when a merchant is paid and when new a merchant onboard"

Reply:
:shrug:

Reply:
Ok I'm making the button checked by default! So sinister of us...